x/payments/internal/types/params.go

```diff
@@ -133,18 +97,218 @@ func (p Params) String() string {
 	)
 }
```

Comment:
`ValidateParams` was not yet removed for project module.

Reply:
ValidateParams removed